lib/ansible/config/base.yml

```diff
@@ -1059,50 +1068,6 @@ DEFAULT_SU:
   - {key: su, section: defaults}
   type: boolean
   yaml: {key: defaults.su}
-DEFAULT_SUDO:
-  default: False
-  deprecated:
-    why: In favor of Ansible Become, which is a generic framework
-    version: "2.9"
```

Comment:
It says this will be removed in 2.9 not 2.8 so sounds like this is being removed a version too early.

Reply:
Ahh I see they have been added to the sudo plugin, ignore me here.